Source/Core/Core/HLE/HLE_VarArgs.h

```diff
@@ -15,13 +16,13 @@ namespace HLE::SystemVABI
 {
 // SFINAE
```

Comment:
Is this comment still applicable? [Cppreference](https://en.cppreference.com/w/cpp/language/sfinae#Alternatives) says concepts are an alternative to "Substitution Failure Is Not An Error".